README.rst

```diff
@@ -5,9 +5,108 @@ Python language bindings for ev3dev
     :target: https://travis-ci.org/rhempel/ev3dev-lang-python
 .. image:: https://readthedocs.org/projects/python-ev3dev/badge/?version=latest
     :target: http://python-ev3dev.readthedocs.org/en/latest/?badge=latest
-    :alt: Documentation Statu
+    :alt: Documentation Status
 
-This is a python library implementing unified interface for ev3dev_ devices.
+A Python library implementing unified interface for ev3dev_ devices.
 
```

Comment:
Thks, I have reorganized this a bit to make it easier for users to simply cut/paste the code (hiding the prompt).